packages/react-native/ReactAndroid/src/main/jni/react/jni/JSLoader.cpp

```diff
@@ -88,10 +88,13 @@ loadScriptFromAssets(AAssetManager* manager, const std::string& assetName) {
   }
 
   throw std::runtime_error(folly::to<std::string>(
-      "Unable to load script. Make sure you're "
-      "either running Metro (run 'npx react-native start') or that your bundle '",
-      assetName,
-      "' is packaged correctly for release."));
+      "Unable to load script.\n\n"
+      "Make sure you're running Metro or that your "
+      "bundle '", assetName, "' is packaged correctly for release.\n\n"
+      "The device must be on the same Wi-Fi network as your laptop to connect to Metro.\n\n"
+      "To use USB instead, shake the device to open the Dev Menu and set "
+      "the bundler location to \"localhost:8081\" and run:\n"
+      "  adb reverse tcp:8081 tcp:8081"));
```

Comment:
nice :)

Reply:
Ah, since we're here still, can we substitute "laptop" for "computer" too?

Also I'm not 100% on how line wrapping should work in Kotlin, but maybe these could be rebalanced? :)

Reply:
@huntie when you import it, the formatter will take care of it

Reply:
I replaced "laptop" with "computer". Not sure about how to rebalance this better though.